scripts/cnv_wdl/cnv_common_tasks.wdl

```diff
@@ -224,6 +273,7 @@ task CollectCounts {
     output {
         String entity_id = base_filename
         File counts = counts_filename
+        File counts_index = if enable_indexing_ then "${base_filename}.${counts_index_filename_extension}" else "/dev/null"
```

Comment:
Not sure if there is a better way to handle optional outputs...can reviewer check?